### ISSUE TITLE
Ignore Safari pointer events on keyboard navigation 

### ISSUE THREAD

**Issue by pacocoursey** [pacocoursey/cmdk#51]
Safari will fire pointer events when navigating via the keyboard, annoyingly causing the active index to jump back up to where the cursor is resting. Instead, these should be ignored when caused via keyboard navigation. 

Could be solved by #49

**Comment by grundmanise** [comment]
To resolve this issue in Safari while keeping pointer selection enabled, I added a check for mouse movement in the item's mouse move handler as follows:

```
    function handleMouseMove(e: React.MouseEvent) {
      const isMoved = e.movementX !== 0 || e.movementY !== 0;
      if (!(disabled || context?.getDisablePointerSelection()) && isMoved) {
        select();
      }
    }
```

Hope that helps someone.
